receiver/elasticapmreceiver/internal/mappers/intakeV2ToDerivedFields.go

```diff
@@ -41,14 +41,14 @@ func SetDerivedFieldsForTransaction(event *modelpb.APMEvent, attributes pcommon.
 	attributes.PutBool("transaction.root", event.ParentId == "")
 	attributes.PutStr("transaction.type", event.Transaction.Type)
 	attributes.PutStr("transaction.result", event.Transaction.Result)
-	attributes.PutInt("transaction.duration.us", int64(event.Event.Duration))
+	attributes.PutInt("transaction.duration.us", int64(event.Event.Duration/1_000))
```

Comment:
As you mention in the PR description, the attribute name is misleading here, since it should be `.ms`?
Is it possible kibana would accept `.ms`?

Reply:
Unfortunately Kibana [expects `us`](https://github.com/search?q=repo%3Aelastic%2Fkibana+transaction.duration.us&type=code); [`ms`](https://github.com/search?q=repo%3Aelastic%2Fkibana+transaction.duration.ms&type=code) seems to be unknown to Kibana. 

From what I know `us` is a common abbreviation for `μs` in case of microseconds. [Wikipedia is](https://en.wikipedia.org/wiki/Microsecond) also aligned with that.